tools/c7n_gcp/c7n_gcp/resources/kms.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2019 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+
+from c7n_gcp.provider import resources
+from c7n_gcp.query import QueryResourceManager, TypeInfo, ChildResourceManager, ChildTypeInfo
+
+
+@resources.register('kms-location')
+class KmsLocation(QueryResourceManager):
```

Comment:
shouldn't be a resource

Reply:
its a static list of strings corresponding to regions, it really doesn't belong as a resource, which would mean extra api calls and round trips for a static list.

Reply:
Ok, I see, removed the resource.